src/main/java/edu/tamu/cap/model/Schema.java

```diff
@@ -63,27 +84,39 @@ public void setAbbreviation(String abbreviation) {
     }
 
     public String getNamespace() {
-		return namespace;
-	}
-
-	public void setNamespace(String prefix) {
-		this.namespace = prefix;
-	}
-
-	public List<Property> getProperties() {
-		return properties;
-	}
-
-	public void setProperties(List<Property> properties) {
-		this.properties = properties;
-	}
-	
-	public void addProperty(Property property) {
-		this.properties.add(property);
-	}
-	
-	public void removeProperty(Property property) {
-		this.properties.remove(property);
-	}
-	
+        return namespace;
+    }
+
+    public void setNamespace(String prefix) {
+        this.namespace = prefix;
+    }
+    
+    public Set<String> getNamespaces() {
+        return namespaces;
+    }
+
+    public List<Property> getProperties() {
+        return properties;
+    }
+    
+    private void setProperties(List<Property> properties) {
+        this.properties = properties;
+    }
+
+    public void addProperties(List<Property> properties) {
```

Comment:
When deserializing this will not be called. Only setProperties will be called and will not have and namespaces.